.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--- Provide a general summary of your changes in the title above -->
+
+## Purpose
```

Comment:
Purpose vs Motivation and Context : as a bug reporting user I am now going to stare at the page wondering what the difference is between these two categories are. 🤕